new-lamassu-admin/src/pages/Transactions/Transactions.js

```diff
@@ -66,12 +103,23 @@ const GET_TRANSACTIONS = gql`
 const Transactions = () => {
   const classes = useStyles()
   const history = useHistory()
-  const { data: txResponse, loading } = useQuery(GET_TRANSACTIONS, {
-    variables: {
-      limit: NUM_LOG_RESULTS
-    },
-    pollInterval: 10000
-  })
+  const chipClasses = useChipStyles()
+
+  const [filters, setFilters] = useState([])
+  const { data: filtersResponse, loading: loadingFilters } = useQuery(
+    GET_TRANSACTION_FILTERS
+  )
+  const [filteredTransactions, setFilteredTransactions] = useState([])
+  const [variables, setVariables] = useState({ limit: NUM_LOG_RESULTS })
+  const { data: txResponse, loading: loadingTransactions, refetch } = useQuery(
+    GET_TRANSACTIONS,
+    {
+      variables,
+      onCompleted: data =>
+        setFilteredTransactions(R.path(['transactions'])(data)),
+      pollInterval: 10000
```

Comment:
this is gonna give use the bug where it refreshes every 10 seconds on the login page after logging out, like the other day
change the code to be like I did at src/components/layout/Header